api/v1alpha1/envoyproxy_types.go

```diff
@@ -68,6 +68,12 @@ type EnvoyProxySpec struct {
 	// +optional
 	Concurrency *int32 `json:"concurrency,omitempty"`
 
+	// EndpointRoutingDisabled can be set to true to use
+	// the Service Cluster IP for routing to the backend
+	// instead.
+	// +optional
+	EndpointRoutingDisabled *bool `json:"endpointRoutingDisabled,omitempty"`
```

Comment:
in the API, can we call this `RoutingType` , creating two enums `Service` and `Endpoint` and defaulting to `Endpoint` ?